### DISCUSSION TITLE
Is mmPretrain not support wandbHook?

### DISCUSSION THREAD

**Discussion by khh0106** [open-mmlab/mmclassification#1491] [Q&A]
in [Doc](https://mmpretrain.readthedocs.io/en/latest/migration.html#dataset-settings)  
I show the command "The MMClsWandbHook in hooks package is waiting for implementation."  
when I use wandbHook?

**Comment by khh0106** [comment]
I find config params in MMEngine
```python
visualizer = dict(type='UniversalVisualizer', 
                  vis_backends=[
                      dict(
                          type='WandbVisBackend', 
                          init_kwargs=dict(project=''))])
```

**Reply by Ezra-Yu** [reply]
This aspect of our documentation is not well represented, do you have suggestions for improvement, or can you just improve our usage documentation and create a PR, where more people can be helped. Thanks.
